src/main/java/com/example/dto/UserDto.java

```diff
@@ -0,0 +1,68 @@
+package com.example.dto;
+
```

Comment:
JD and final?

Reply:
@Vlada0612 I see no need at this stage to apply Java Docs to DTOs

Reply:
jd is necessary for dto for understanding of appointment of this dto, and it's necessary always. And it used on my previous project always and permanently.